test/e2e/specs/editor/various/block-bindings/post-meta.spec.js

```diff
@@ -322,7 +322,7 @@ test.describe( 'Post Meta source', () => {
 			// Check the post meta fields are not visible.
 			const globalField = page
 				.getByRole( 'menuitemradio' )
-				.filter( { hasText: 'text_custom_field' } );
+				.filter( { hasText: 'Text custom field' } );
```

Comment:
I updated some tests here due to the label property now being supplied in `packages/e2e-tests/plugins/block-bindings.php`.

Reply:
Related to this, I believe `text_custom_field` in the following test should also be rendering as `Text custom field`, but it's not and I'm not sure why:

https://github.com/WordPress/gutenberg/blob/1bb27791da0c61b7528fe7fbf754a828178cf03b/test/e2e/specs/editor/various/block-bindings/post-meta.spec.js#L332-L355

Reply:
It is because that test is part of the Custom Template tests, which doesn't support post meta yet (because we aren't able to get only the fields that apply to any post type).

Thinking about it, maybe it is better to create a new "String custom field"? We were using "text custom field" to test that the key is shown when there is no label, and that's a valid use case.